components/aws/sagemaker/tests/integration_tests/Dockerfile

```diff
@@ -39,6 +39,7 @@ ENV MINIO_LOCAL_PORT=9000
 ENV KFP_NAMESPACE="kubeflow"
 
 RUN mkdir pipelines
-COPY ./ ./pipelines/
 
-ENTRYPOINT [ "/bin/bash", "./pipelines/components/aws/sagemaker/tests/integration_tests/scripts/run_integration_tests" ]
\ No newline at end of file
+ENTRYPOINT [ "/bin/bash" ]
+
+CMD [ "" ]
```

Comment:
We might need to keep this the same for our CI system. I'll let you know when they run through